scripts/upgrade-zulip-from-git

```diff
@@ -55,6 +55,7 @@ get_deployment_lock(error_rerun_script)
 
 try:
     deploy_path = make_deploy_path()
+    deploy_branch = 'deploy-' + deploy_path.split('/')[-1]
```

Comment:
This was the simplest way I felt to get the same timestamp as deploy_path instead of 
(returning tuples)/(making a dict)/(make a new function like make_deploy_path_return_timestamp) just for this one case.
Please let me know if this needs to be changed.